src/frontend/apps/standalone_site/craco.config.js

```diff
@@ -0,0 +1,31 @@
+const { getLoader, loaderByName } = require('@craco/craco');
+const {
+  appPackages: packages,
+  alias,
+  moduleNameMapper,
+} = require('marsha-config');
+
+module.exports = {
```

Comment:
Can you explain with a comment what you are modifying here from the original webpack config in the `create-react-app` package ?

Reply:
https://github.com/openfun/marsha/pull/2150/commits/b89c81b592c8b132de84fce878624b8649366126